src/borg/archiver.py

```diff
@@ -2626,34 +2626,56 @@ def do_break_lock(self, args, repository):
-        specify the backup roots (starting points) and patterns for inclusion/exclusion.
-        A root path starts with the prefix `R`, followed by a path (a plain path, not a
-        file pattern). An include rule starts with the prefix +, an exclude rule starts
-        with the prefix -, an exclude-norecurse rule starts with !, all followed by a pattern.
+        A more general and easier to use way to define filename matching patterns
+        exists with the ``--pattern`` and ``--patterns-from`` options. Using
+        these, you may specify the backup roots, default pattern styles and
+        patterns for inclusion/exclusion.
 
-        .. note::
+        Root path prefix ``R``
+            A root path starts with the prefix `R`, followed by a path (a plain
+            path, not a file pattern). Use this prefix to move root paths from
+            command line arguments to a patterns file.
 
-            Via ``--pattern`` or ``--patterns-from`` you can define BOTH inclusion and exclusion
-            of files using pattern prefixes ``+`` and ``-``. With ``--exclude`` and
-            ``--exclude-from`` ONLY excludes are defined.
+        Pattern style prefix ``P``
+            To change the default pattern style, use the `P` prefix, followed by
+            the pattern style abbreviation (`fm`, `pf`, `pp`, `re`, `sh`). All
+            patterns following this line will use this style until another style
+            is specified.
 
-        Inclusion patterns are useful to include paths that are contained in an excluded
-        path. The first matching pattern is used so if an include pattern matches before
-        an exclude pattern, the file is backed up. If an exclude-norecurse pattern matches
-        a directory, it won't recurse into it and won't discover any potential matches for
-        include rules below that directory.
+        Include pattern prefix ``+``
+            Use the prefix `+`, followed by a pattern, to define include files.
+            Include patterns can be used with ``--patterns`` and ``--patterns-from``
+            only.
 
-        .. note::
+        Exclude pattern prefix ``-``
+            Use the prefix `-`, followed by a pattern, to define an exclusion
+            for ``--patterns`` / ``--patterns-from`` and ``--exclude`` /
+            ``--exclude-from``.
+
+        Exclude no-recurse pattern prefix ``!``
+            Use the prefix `!`, followed by a pattern, to define an exclusion
+            that does not recurse into subdirectories.
+
+        Inclusion patterns are useful to include paths that are contained in an
+        excluded path. The first matching pattern is used. so if an include
+        pattern matches before an exclude pattern, the file is backed up. If an
+        exclude-norecurse pattern matches a directory, it won't recurse into it
+        (which saves time), but won't discover any potential matches for include
+        rules below that directory.
```

Comment:
change order:

1. exclude
2. exclude no-recurse
3. include

and then explain why include is useful (maybe even within the include explanation).

Reply:
ok